package.json

```diff
@@ -17,15 +17,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.147",
-    "@companieshouse/node-session-handler": "^5.0.1",
-    "@companieshouse/structured-logging-node": "^2.0.1",
+    "@companieshouse/api-sdk-node": "^2.0.103",
+    "@companieshouse/node-session-handler": "^4.1.9",
+    "@companieshouse/structured-logging-node": "^1.0.8",
     "cookie-parser": "1.4.5",
```

Comment:
shouldn't we be using the latest versions of CH packages here?

Reply:
ah ok i took this branch before my previous feature branch gone IN which was CC-1149 thats why its reverting all changes